hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/TableSnapshotInputFormat.java

```diff
@@ -94,12 +93,6 @@ public TableSnapshotRegionSplit(TableSnapshotInputFormatImpl.InputSplit delegate
       this.delegate = delegate;
     }
 
-    public TableSnapshotRegionSplit(TableDescriptor htd, RegionInfo regionInfo,
```

Comment:
The class is marked as IA.Public, so you can not delete a public method from it directly. You need to make it deprecated for a whole major release cycle before deleteing.